package.json

```diff
@@ -7,7 +7,7 @@
     "test": "test"
   },
   "scripts": {
-    "test": "npm run build:ts && tsc -p test/tsconfig.json && tap --ts \"test/**/*.test.ts\"",
+    "test": "NODE_ENV=test tap --ts \"test/**/*.test.ts\" --coverage-report=text-lcov | tap-nyc --check-coverage --lines 95 --branches 100 --statements 95",
```

Comment:
What are the 95 and 100 numbers?
Usually we don't hard code numbers like this...

Reply:
yes, you're right. edited.